src/Sylius/Bundle/AdminBundle/Resources/config/app/sylius/resources/customer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+use Sylius\Bundle\AdminBundle\Form\Type\CustomerType;
+use Sylius\Resource\Metadata\BulkDelete;
+use Sylius\Resource\Metadata\Create;
+use Sylius\Resource\Metadata\Index;
+use Sylius\Resource\Metadata\Operations;
+use Sylius\Resource\Metadata\ResourceMetadata;
+use Sylius\Resource\Metadata\Show;
+use Sylius\Resource\Metadata\Update;
+
+return (new ResourceMetadata())
+    ->withRoutePrefix('/%sylius_admin.path_name%')
+    ->withClass('%sylius.model.customer.class%')
+    ->withSection('admin')
+    ->withTemplatesDir('@SyliusAdmin/shared/crud')
+    ->withFormType(CustomerType::class)
+    ->withRouteCondition("not context.isSyliusRoutingBcLayerEnabled('admin_customer')")
+    ->withOperations(new Operations(operations: [
+        new Create(
+            routeName: '_sylius_admin_customer_create',
+            redirectToRoute: 'sylius_admin_customer_update',
+        ),
+        new Update(
+            routeName: '_sylius_admin_customer_update',
+            redirectToRoute: 'sylius_admin_customer_update',
+        ),
+        new BulkDelete(
```

Comment:
technically, legacy routing has a bulk delete operation... I'm wondering if we should add it or not.